_episodes/016-twir-340.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "This Week in Rust - Issue 340"
+date: 2020-05-27T14:30:00Z
+file:  https://audio.rustacean-station.org/file/rustacean-station/twir-2020-05-26.mp3
```

Comment:
I think CI is choking on the extra whitespace between the `:` and the URL here.

Reply:
Ah! I'll get that fixed!